app/Http/Controllers/RefreshedCollectionController.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Jobs\RefreshWalletCollections;
+use App\Support\Queues;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+
+class RefreshedCollectionController extends Controller
+{
+    public function store(Request $request): RedirectResponse
+    {
+        abort_unless($request->wallet()->canRefreshCollections(), 403);
+
+        RefreshWalletCollections::dispatch($request->wallet())->onQueue(Queues::WALLETS);
+
+        return back()->toast(
```

Comment:
@crnkovic Not sure exactly about this, but I'm still getting redirected to the landing page whenever I hit the refresh button. Does it not happen for you?

Reply:
This is how it looks for me.

https://github.com/ArdentHQ/dashbrd/assets/6536260/e280a21f-a27f-4f1a-8f08-2099d6413d2c